crypto/bn/gcd.c

```diff
@@ -484,14 +494,17 @@ static BIGNUM *BN_mod_inverse_no_branch(BIGNUM *out, int *out_no_inverse,
      *      sign*Y*a  ==  A   (mod |n|)
      */
 
-    /* Turn BN_FLG_CONSTTIME flag on, so that when BN_div is invoked,
+    /* Ensure BN_FLG_CONSTTIME flag is on, so that when BN_div is invoked,
      * BN_div_no_branch will be called eventually.
      */
-    pA = &local_A;
-    BN_with_flags(pA, A, BN_FLG_CONSTTIME);
+    assert(BN_get_flags(A, BN_FLG_CONSTTIME) != 0);
+    if (BN_get_flags(A, BN_FLG_CONSTTIME) == 0) {
+      OPENSSL_PUT_ERROR(BN, 100000);  // FIXME: what error code?
+      goto err;
+    }
```

Comment:
Ditto here, since A is a local variable you can just move the `BN_set_flags` here.